spec/system/nested_attributes_spec.rb

```diff
@@ -80,6 +80,77 @@
   end
 end
 
+RSpec.describe "HTML attributes for 'abyme-fields' & add/remove association", type: :system do
```

Comment:
Ici t'es pas obligé de rappeler RSpec. Tu peux soit : 
- directement écrire `describe "what this test is doing"`
- soit comme je l'ai fait au-dessus, utiliser le keyword `context`